dirs/dirs.go

```diff
@@ -167,6 +171,8 @@ var (
 type SnapDirOptions struct {
 	// HiddenSnapDataDir determines if the snaps' data is in ~/.snap/data instead of ~/snap
 	HiddenSnapDataDir bool
+	// UseExposedDir determines if ~/Snap is in use.
+	UseExposedDir bool
```

Comment:
this should have a name that matches a bit more the ones we pick in the snap package